doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -393,6 +393,9 @@ The following example script illustrates all these features:
     { trivial. }
     assumption.
 
+  .. coqtop:: none
+
+    Abort.
```

Comment:
Note: this one is already fixed by #8266 and this change will create a conflict when #8266 is merged.